eng/testing/debug-dump-template.md

```diff
@@ -1,127 +1,145 @@
-# Debugging a CI dump
+# Get the dump
 
-This document describes how to debug a CI/PR test dump by downloading assets from helix, using a dotnet tool called `runfo`.
+Click the link to the dump on the `Helix Test Logs` tab in Azure DevOps. This is the same place you got these instructions from.
 
-## What is runfo?
+# Get the Helix payload
 
-Runfo is a dotnet global tool that helps get information about helix test runs and azure devops builds. For more information see [this](https://github.com/jaredpar/runfo/tree/master/runfo#runfo)
+[Runfo](https://github.com/jaredpar/runfo/tree/master/runfo#runfo) helps get information about helix test runs and azure devops builds. We will use it to download the payload and symbols:
+```script
+dotnet tool install --global runfo
+dotnet tool update --global runfo
+```
+If prompted, open a new command prompt to pick up the updated PATH.
+```script
+# On Windows
+runfo get-helix-payload -j %JOBID% -w %WORKITEM% -o %WOUTDIR%
+# On Linux and macOS
+runfo get-helix-payload -j %JOBID% -w %WORKITEM% -o %LOUTDIR%
```

Comment:
`$JOBID` instead of `%JOBID%` on Linux/macOS?
````suggestion
```cmd
:: On Windows
runfo get-helix-payload -j %JOBID% -w %WORKITEM% -o %WOUTDIR%
```
```sh
# On Linux and macOS
runfo get-helix-payload -j $JOBID -w $WORKITEM -o $LOUTDIR
````

Reply:
Thanks, but they are'nt environment variables
https://github.com/dotnet/runtime/pull/46493/files#diff-fd8bb0da491f288f6e268f835a49b14126ba3f1a15ab6971ddc76dd480b81b11R97